contracts/contracts/ProjectRegistry.sol

```diff
@@ -124,11 +124,11 @@ contract ProjectRegistry is Initializable {
      * @param owner Address of new Owner
      */
     function removeProjectOwner(uint256 projectID, address prevOwner, address owner) external onlyProjectOwner(projectID) {
-        require(owner != address(0) && owner != OWNERS_LIST_SENTINEL, "bad owner");
+        require(owner != address(0) && owner != OWNERS_LIST_SENTINEL, "invalid owner");
 
         OwnerList storage owners = projectsOwners[projectID];
 
-        require(owners.list[prevOwner] == owner, "bad prevOwner");
+        require(owners.list[prevOwner] == owner, "prevOwner must equal owner");
```

Comment:
@schultztimothy `prevOwner must equal owner` might be misleading cause in this case if `owners.list[prevOwner]` is `!==  owner` it means `prevOwner` is actually not the previous owner. maybe we can have something like `PR001` as advised in the review so that we can explain it better?

Reply:
Good call!